test/e2e/utils/asserts.go

```diff
@@ -17,6 +17,7 @@ func LogError(err error) {
 // ExpectNoError checks if an error exists, and if so halts execution
 func ExpectNoError(err error) {
 	if err != nil {
+		LogError(err)
```

Comment:
I think this is redundant as we already log the error in `kubernetes.go#CleanNamespaceAndLogWithPanic` after the panic is thrown.

Reply:
You're right. In fact, I have the same results. I'm removing it. Thanks